modules/sfp_ipqualityscore.py

```diff
@@ -156,21 +160,74 @@ def handleEvent(self, event):
         fraudScore = data.get('fraud_score')
         recentAbuse = data.get('recent_abuse')
         botStatus = data.get('bot_status')
+        malicious = False
+        maliciousDesc = ""
+
         if fraudScore >= self.opts['abuse_score_threshold'] or recentAbuse or botStatus:
             evt = SpiderFootEvent("RAW_RIR_DATA", str(data), self.__name__, event)
             self.notifyListeners(evt)
+            malicious = True
+            maliciousDesc = f"IPQualityScore [ {eventData} ]\n"
+
+        if eventName == "PHONE_NUMBER":
```

Comment:
I believe the API also indicates the phone number type, or at least if it's disposable? That way this module could return `PHONE_NUMBER_TYPE`, and we could also add `PHONE_NUMBER_DISPOSABLE` as a type, just as we have for emails.

Reply:
Don't see anything like this in the response